src/stores/SearchStore.ts

```diff
@@ -375,12 +374,27 @@ export class SearchStore {
 	};
 
 	private onChannelResponse = (ev: Event) => {
+		this.setCurrentSearchTimestamp(ev);
```

Comment:
currently we are parsing event twice, which is unnecessary. lets handle 'keep_alive' event here also
parse event, add type guards and act accordingly